odk2odm/odk_requests.py

```diff
@@ -95,86 +120,102 @@ def odata_submissions(base_url, aut, projectId, formId):
     each dict is a single submission with the form question names as keys.
     """    
     url = f'{base_url}/v1/projects/{projectId}/forms/{formId}.svc/Submissions'
-    submissions = requests.get(url, auth = aut)
+    submissions = requests.get(url, auth=aut)
     return submissions
 
 
 def attachment_list(base_url, aut, projectId, formId, instanceId):
     """Fetch an individual media file attachment."""
     url = f'{base_url}/v1/projects/{projectId}/forms/{formId}/submissions/'\
         f'{instanceId}/attachments'
-    return requests.get(url, auth = aut)
+    return requests.get(url, auth=aut)
 
 
 def attachment(base_url, aut, projectId, formId, instanceId, filename):
     """Fetch a specific attachment by filename from a submission to a form."""
     url = f'{base_url}/v1/projects/{projectId}/forms/{formId}/submissions/'\
         f'{instanceId}/attachments/{filename}'
-    return requests.get(url, auth = aut)
+    return requests.get(url, auth=aut)
 
 # POST 
 def create_project(base_url, aut, project_name):
     """Create a new project on an ODK Central server"""
     url = f'{base_url}/v1/projects'
-    return requests.post(url, auth = aut, json = {'name': project_name})
+    return requests.post(url, auth=aut, json={'name': project_name})
 
-def create_app_user(base_url, aut, projectId, app_user_name = 'Surveyor'):
+def create_app_user(base_url, aut, projectId, app_user_name='Surveyor'):
     """
     Create a new project on an ODK Central server
 
     Atm. you can create multiple app users with the same name, should this be possible, or give an error? 
     """
     url = f'{base_url}/v1/projects/{projectId}/app-users'
-    return requests.post(url, auth = aut, json = {'displayName': app_user_name})
+    return requests.post(url, auth=aut, json={'displayName': app_user_name})
 
 
-def give_access_app_users(base_url, aut, projectId):
```

Comment:
The reason why I had the function to give access to all the app users was to be able to just create an app user without pushing any buttons to give them access. The way I understood it, all the app users should have access to all the forms anyway. Also, when creating the app user, it doesn't return the id, so we would have to make the app user names unique to be able to search the ids based on the name.

Reply:
It makes a lot of sense. I would then keep these smarter functionalities separate from the basic request recipes, and refer to those. We can think of a separate set of wrapper functions in a separate file. Then the odk_requests.py can only contain functions that return a http response.